pages/model/[uuid].tsx

```diff
@@ -62,6 +63,14 @@ const Alert = React.forwardRef<HTMLDivElement, AlertProps>((props, ref) => (
   <MuiAlert elevation={6} ref={ref} variant='filled' {...props} />
 ))
 
+function ConfirmationFeedbackText({ status, statusText }: { status: number; statusText: string }) {
+  return (
+    <Alert severity='error' sx={{ m: 2, width: 400 }}>
```

Comment:
You also should steer clear of using fixed widths/heights where possible as this won't work on every screen size

Reply:
I think this was me - I added a width because by default the window looked very small, perhaps a min-width would be better, but I'm not against removing it altogether.

Reply:
By default the Alert is using the full width, so we have removed the `width` property